src/codeyCommand.ts

```diff
@@ -10,72 +33,256 @@ export type SapphireMessageExecuteType = (
   client: SapphireClient<boolean>,
   // Message is for normal commands, ChatInputInteraction is for slash commands
   messageFromUser: Message | SapphireCommand.ChatInputInteraction,
-  initialMessageFromBot?: SapphireMessageRequest
+  initialMessageFromBot?: SapphireMessageRequest,
+  // Command arguments
+  args?: CodeyCommandArguments
```

Comment:
Without any arguments, it looks like `args` is still defined to be `{}` and then passed into `commandDetails.executeCommand`, so I think `args` can be required instead. Since required parameters must come before optional parameters, I guess `initialMessageFromBot` and `args` should be switched around.